src/rp2_common/pico_bootrom/include/pico/bootrom.h

```diff
@@ -1074,9 +1074,15 @@ int rom_add_flash_runtime_partition(uint32_t start_offset, uint32_t size, uint32
 /*! \brief  Pick A/B partition for a separate partition
  * \ingroup pico_bootrom
  *
- * This will perform extra checks to prevent disrupting a main image TBYB, and return errors
+ * This will call `rom_pick_ab_partition` with the current `flash_update_boot_window_base`, while performing extra checks to prevent disrupting a main image TBYB.
+ * It requires the same minimum workarea size as `rom_pick_ab_partition`.
  * 
- * Also checks that the chosen partition contained a valid image
+ * For example, if an `explicit_buy` is pending then calling `pick_ab_partition` would normally clear the saved `flash_erase_addr` so the required erase would not
```

Comment:
This doxygen mentions both `rom_pick_ab_partition` and `pick_ab_partition` - are they referring to the same function or are they different?